db/views/rejection_percentage_by_prison_and_calendar_weeks_v02.sql

```diff
@@ -0,0 +1,65 @@
+SELECT rejected.prison_name,
+       'total' AS reason,
+       rejected.year AS year,
+       rejected.week AS week,
+       ROUND((rejected_count::numeric / booked_count * 100),2) AS percentage
+FROM (SELECT prisons.name AS prison_name,
+             count(*) AS rejected_count,
+             extract(isoyear from visits.created_at)::integer AS year,
+             extract(week from visits.created_at)::integer AS week
+      FROM visits
+      INNER JOIN prisons ON prisons.id = visits.prison_id
+      INNER JOIN rejections ON rejections.visit_id = visits.id
```

Comment:
Do you need to join on rejections? As far as I can see visits and prisons should be enough?